db/controllers/reviews.controller.js

```diff
@@ -2,11 +2,15 @@ const { fetchReview } = require("../models/reviews.model");
 
 exports.getReview = (request, response, next) => {
   const reviewToFetch = request.params;
-  fetchReview(reviewToFetch)
-    .then((fetchedReview) => {
-      response.status(200).send(fetchedReview);
-    })
-    .catch((err) => {
-      next(err);
-    });
+  if (reviewToFetch.review_id.match(/([0-9])+/)) {
+    fetchReview(reviewToFetch)
+      .then((fetchedReview) => {
+        response.status(200).send(fetchedReview);
+      })
+      .catch((err) => {
+        if (err.status && err.message) {
+          response.status(err.status).send({ message: err.message });
+        } else next(err);
+      });
+  } else response.status(400).send({ message: "Invalid Review ID" });
```

Comment:
If you're going to do custom error messages then you'll need to use Promise.reject rather than add it in a lengthy catch block. That is what our error handling functions are for.